modules/discourse/main.tf

```diff
@@ -66,3 +69,14 @@ resource "aws_s3_bucket" "discourse-content" {
         project = "discourse"
     }
 }
+
+module "discourse-cdn" {
+  source              = "git://github.com/mozilla/partinfra-terraform-cloudfrontssl.git"
+
+  origin_domain_name  = "${var.fqdn}"
+  origin_path         = "/"
+  origin_id           = "discourse-pull-origin"
+  alias               = "cdn-${var.environment}.discourse.mozilla-community.org"
```

Comment:
We need to refactor DNS the same way I did SSL, so it's best to do this later with everything else.